lib/install/management/appliance.go

```diff
@@ -147,12 +147,16 @@ func (d *Dispatcher) createApplianceSpec(conf *metadata.VirtualContainerHostConf
 	var devices object.VirtualDeviceList
 	var err error
 
+	cfg := make(map[string]string)
+	extraconfig.Encode(extraconfig.MapSink(cfg), conf)
```

Comment:
So change to use extraconfig directly? Is that available through rpctool inside guest os?